static/beta/stage/navigation/rhel-navigation.json

```diff
@@ -541,14 +541,22 @@
           "subtitle": "Red Hat Insights for RHEL",
           "description": "Perform Red Hat-recommended analysis across your Red Hat Enterprise Linux fleet, like upgrade and conversion readiness.",
           "alt_title": [
-              "Tasks",
-              "task",
-              "CentOS conversion",
-              "Pre Conversion analysis",
-              "Pre-Conversion analysis",
-              "Convert to RHEL",
-              "C2R",
-              "CentOS"
+            "Tasks",
+            "tasks"
+            "task",
+            "CentOS conversion",
```

Comment:
Can we also add "centos conversion" and "conversion"? Since the search service sometimes also doesn't work properly with partial matches

Reply:
it also doesn't work correctly with multi-word matches... even now "CentOS conversion" will not return anything. "conversion" will return Tasks correctly. I will add capitalized version of this string. "centos" and its variants are few lines lower.